azure-app-service/massdriver.yaml

```diff
@@ -31,12 +31,16 @@ params:
     - image
     - dns
   properties:
-    application: 
+    command:
+      title: Command
```

Comment:
Would it be more clear for the customer to set up something like this? 
```suggestion
      title: Startup Command
```

Reply:
Yes, I like it